src/actions/index.js

```diff
@@ -144,3 +144,13 @@ export const toggleChat = isOtherChatActive => {
     payload: isOtherChatActive,
   };
 };
+
+export const STREAM_BANNED = Symbol('STREAM_BANNED');
+export const streamBanned = () => dispatch => {
+  //browserHistory.push('/');
+  alert('stream banned or something');
+  dispatch({
+    type: STREAM_BANNED,
+    payload: null,
+  });
+};
```

Comment:
I don't think this addition is used anywhere, since the action was ultimately implemented in `src/actions/websocket.js`.